test/stubs/breath_cycle_stub.c

```diff
@@ -0,0 +1,16 @@
+#include "breath_cycle_stub.h"
+
+int breath_cycle_duration;
+
+int set_default_breath_cycle_duration(){

```

Comment:
do we need a stub for this or can it be direclty include into you piece of code? if you decide to do this we will need to have unit test for the set_default_breath_cycle_duration() funciton too